teamwork/templates/core/matchstats.html

```diff
@@ -10,6 +10,10 @@
 
 {% endblock stylesheets %}
 
+{% block crumb %}
+<li><a href="{% url 'view_matches' %}">My Matches</a></li>
```

Comment:
Breadcrumb for Matchstats should display: "Home > My Matches > Matchstats( with active css class)" currently doesn't show the current window.